Sources/ResultExtensions.swift

```diff
@@ -1,7 +1,41 @@
-import Result
-
 extension Result: SignalProducerConvertible {
 	public var producer: SignalProducer<Success, Failure> {
 		return .init(result: self)
 	}
 }
+
+/// An “error” that is impossible to construct.
+///
+/// This can be used to describe `Result`s where failures will never
+/// be generated. For example, `Result<Int, NoError>` describes a result that
+/// contains an `Int`eger and is guaranteed never to be a `failure`.
+public enum NoError: Swift.Error, Equatable {
```

Comment:
`NoError` can be replaced by `Never`.

- https://github.com/apple/swift-evolution/blob/master/proposals/0215-conform-never-to-hashable-and-equatable.md
- https://github.com/apple/swift/pull/16857

Reply:
Oh that's right!! I'll get rid of `NoError`.